docs/source/distributed.algorithms.join.rst

```diff
@@ -15,3 +16,5 @@ inputs. This page outlines the API of the relevant classes: :class:`Join`,
 
 .. autoclass:: torch.distributed.algorithms.JoinHook
     :members:
+
+.. _Distributed Training with Uneven Inputs Using the Join Context Manager: https://pytorch.org/tutorials/advanced/generic_join.html
```

Comment:
I assume this won't be rendered and will only be used by the reference on line 9 as a hyperlink?